packages/replay/src/integration.ts

```diff
@@ -130,6 +155,11 @@ export class Replay implements Integration {
       // collect fonts, but be aware that `sentry.io` needs to be an allowed
       // origin for playback
       collectFonts: true,
+      errorHandler: (err: Error) => {
+        // @ts-ignore Set this so that replay SDK can ignore errors originating from rrweb
+        err.__rrweb__ = true;
```

Comment:
We should try-catch all of this, like:

```js
 try {
        error.__rrweb__ = true;
      } catch {
        // ignore errors here
      }
```

To avoid weird stuff there 😅